src/control/lib/daos/system_prop.go

```diff
@@ -0,0 +1,346 @@
+//
+// (C) Copyright 2022 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+package daos
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/daos-stack/daos/src/control/build"
+	"github.com/daos-stack/daos/src/control/common"
+)
+
+// BoolPropVal is a boolean property value.
+type BoolPropVal bool
+
+// NewBoolPropVal returns a new BoolPropVal initialized to a default value.
+func NewBoolPropVal(defVal bool) *BoolPropVal {
+	pv := BoolPropVal(defVal)
+	return &pv
+}
+
+func (pv *BoolPropVal) Handler(val string) error {
```

Comment:
Hm, "Handler" is a little non-intuitively named. "SetValue" may be a clearer name.

Reply:
Yeah... This stuff is at least loosely based on the [work](https://github.com/daos-stack/daos/blob/master/src/control/cmd/daos/property.go#L604) I did in the daos tool. When I did that, we didn't have the `lib/daos` package, so a lot of non-UI stuff wound up in `cmd/daos`. At some point I'd like to look at unifying all of this stuff across container/pool/system properties/attributes and having a shared codebase in `lib/daos`.